.storybook/preview.ts

```diff
@@ -13,7 +12,14 @@ const preview: Preview = {
     },
     viewport: {
       viewports: {
-        ...MINIMAL_VIEWPORTS,
+        mobile: {
+          name: 'Mobile',
+          styles: {
+            width: '414px',
+            height: '736px',
+          },
+          type: 'mobile',
+        },
```

Comment:
Consider to add these to PR description as it does not seem to be really part of the ticket per se.